tests/app/service/test_archived_service.py

```diff
@@ -57,7 +57,7 @@ def test_deactivating_service_changes_name_and_email(client, sample_service):
     archived_service = dao_fetch_service_by_id(sample_service.id)
 
     assert archived_service.name == "_archived_2018-07-07_Sample service"
-    assert archived_service.normalised_service_name == "archived20180707sample.service"
+    assert archived_service.email_sender_local_part == "archived20180707sample.service"
```

Comment:
ew i don't like this 🙃

Reply:
i hadn't really thought about this. the _motivation_ for changing `_normalised_service_name` is to make sure the old service name doesn't conflict with the old service name now that it's inactive.

it's a bit weird that this'll get updated if they use their service name, but will get left untouched if they have a custom email sender defined.

couple of options (there may be other things we could do)

* change _normalised_service_name by accessing the column directly and bypassing the property - that way `email_sender_local_part` would be unchanged
* accept this as a quirk. we'd need to make sure that if/when we unarchive, the script we use correctly restores `email_sender_local_part` (which it will do automatically if it uses the `service.name` property to unarchive the service)

Reply:
> change _normalised_service_name by accessing the column directly and bypassing the property - that way email_sender_local_part would be unchanged

I think this is sort of OK.

I'd also be pro us thinking about just not changing the service name/various attributes other than `state` at all when archiving. It feels like a weird mutation for us to do 🤔 There are probably UI considerations though if we remove that semi-obvious marker. We should probably have an "archived services" org view for example, rather than just listing them in the live/trial services view.

Reply:
(can be followed-up separately)